docs/manual/grelfunctions.md

```diff
@@ -35,6 +35,30 @@ Uses the logical operator NOT on a boolean to output a boolean. For example, `no
 
 Uses the logical operator XOR (exclusive-or) on two or more booleans to output a boolean. Evaluates multiple statements, then returns true if only one of them is true. For example, `(1 < 3).xor(1 < 7)` returns false because more than one of the conditions is true.
 
+### Operators {#operators}
+
+#### Arithmetic Operators {#arithmetic-operators}
+
```

Comment:
Move (or copy?) the division documentation from the Math Functions page to here.